package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@finos/git-proxy",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

```suggestion
  "version": "1.1.0",
```
@coopernetes @maoo (cc)